akka23-common/src/main/scala/com/typesafe/conductr/lib/akka/ConnectionHandler.scala

```diff
@@ -147,6 +135,10 @@ class ConnectionHandler extends AbstractConnectionHandler {
     requestF.map { request =>
       request.addHeader(`User-Agent`(UserAgent))
       request.addHeader(Host(url.getHost, url.getPort))
+      payload.getRequestHeaders.foldLeft(request) { (request, entry) =>
+        val (header, headerValue) = entry
+        request.addHeader(RawHeader(header, headerValue))
+      }
```

Comment:
Given that we've taken care of Akka23/24, shouldn't we also address pure Scala, Play and Lagom?